showcase/app/components/mock/app/header/app-header.gts

```diff
@@ -51,22 +52,25 @@ export default class MockAppHeaderAppHeader extends Component<MockAppHeaderAppHe
           @href="#"
         />
       </:logo>
-      <:globalActions>
+      <:globalActions as |actions|>
         {{#if this.showOrgPicker}}
           <HdsDropdown @enableCollisionDetection={{true}} as |dd|>
             <dd.ToggleButton @text={{this.orgPickerLabel}} @icon="org" />
-            <dd.Checkmark>
+            <dd.Checkmark {{on "click" actions.close}}>
```

Comment:
This implementation is nice. Seems very simple & convenient for consumers to use.